test/runtests.jl

```diff
@@ -320,6 +321,15 @@ Unitful.uconvert(U::Unitful.Units, q::QQQ) = uconvert(U, Quantity(q.val, cm))
 
 @testset "Promotion" begin
     @testset "> Unit preferences" begin
+        # Should warn on possible redundant units issue (ms and s)
+        @test_logs (:warn, r"^Preferred units contain complex units") Unitful.preferunits(C/ms)
+        # Test for wacky prefered units functionality
+        Unitful.preferunits(C/s)
+        @test @inferred(upreferred(u"V/m")) == u"kg*m*C^-1*s^-2"
+        @test dimension(upreferred(u"V/m")) == dimension(u"V/m")
```

Comment:
```suggestion
        @test @inferred(upreferred(V/m)) == kg*m*C^-1*s^-2
        @test dimension(upreferred(V/m)) == dimension(V/m)
```
All of these units are imported, so we don’t need the `u"…"` macro.